VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: CCU PIU DCU FC
VAL_TABLE_ DriverButton 1 "ON" 0 "OFF" ;
VAL_TABLE_ CoolingPowerModes 4 "FULL" 3 "HIGH" 2 "MEDIUM" 1 "LOW" 0 "OFF" ;
VAL_TABLE_ SensorStatus 4 "SHORTED" 3 "GROUNDED" 2 "OUT_OF_RANGE_UPPER" 1 "OUT_OF_RANGE_LOWER" 0 "NO_FAULT" ;
VAL_TABLE_ FaultStates 1 "FAULT_ACTIVE" 0 "NO_FAULT" ;
VAL_TABLE_ StartupStates 3 "START" 2 "ON" 1 "ACCESSORY" 0 "LOCK" ;


BO_ 1536 AccelerometerData: 8 PIU
 SG_ YawRate : 32|16@1- (1,0) [0|0] ""  FC
 SG_ AccelLongitudinal : 16|16@1- (1,0) [0|0] ""  FC
 SG_ AccelLateral : 0|16@1- (1,0) [0|0] ""  FC

BO_ 768 LvBatteryData: 8 CCU
 SG_ GlvVoltage : 8|8@1+ (1,0) [0|0] ""  FC
 SG_ GlvCurrent : 0|8@1+ (1,0) [0|0] ""  FC

BO_ 1283 VehicleInfo: 5 FC
 SG_ VehicleSpeed : 24|8@1+ (1,0) [0|255] "km/h"  DCU
 SG_ VcuErrors : 16|8@1+ (1,0) [0|255] ""  DCU
 SG_ VehicleStatus : 32|2@1+ (1,0) [0|3] ""  DCU
 SG_ SocPercentage : 8|7@1+ (1,0) [0|127] "%"  DCU
 SG_ MaximumPowerAvailablePercentage : 0|7@1+ (1,0) [0|127] ""  DCU

BO_ 83 CoolingControllerAnswer: 1 CCU
 SG_ CcuAnswer : 0|8@1+ (1,0) [0|255] ""  FC

BO_ 85 DashboardAnswer: 1 DCU
 SG_ DcuAnswer : 0|8@1+ (1,0) [0|255] ""  FC

BO_ 81 PiuControllerAnswer: 1 PIU
 SG_ PiuAnswer : 0|8@1+ (1,0) [0|255] ""  FC

BO_ 80 PiuControllerQuestion: 1 FC
 SG_ PiuQuestion : 0|8@1+ (1,0) [0|255] ""  PIU

BO_ 84 DashboardQuestion: 1 FC
 SG_ DcuQuestion : 0|8@1+ (1,0) [0|255] ""  DCU

BO_ 82 CoolingControllerQuestion: 1 FC
 SG_ CcuQuestion : 0|8@1+ (1,0) [0|255] ""  CCU

BO_ 1281 DashboardData: 2 DCU
 SG_ GpsVehicleSpeed : 0|12@1+ (0.1,0) [0|409.5] ""  FC

BO_ 1282 TemperatureInfo: 5 FC
 SG_ BatteryTemperature : 32|8@1+ (1,-20) [-20|235] "�C" Vector__XXX
 SG_ RightRearMotorTemperature : 24|8@1+ (1,-20) [-20|235] "�C"  DCU
 SG_ RightFrontMotorTemperature : 16|8@1+ (1,-20) [-20|235] "�C"  DCU
 SG_ LeftRearMotorTemperature : 8|8@1+ (1,-20) [-20|235] "�C"  DCU
 SG_ LeftFrontMotorTemperature : 0|8@1+ (1,-20) [-20|235] "�C"  DCU

BO_ 513 CoolingLoopsData: 7 CCU
 SG_ InverterLoopTempSensorStatus : 30|3@1+ (1,0) [0|7] ""  FC
 SG_ InverterLoopPressureSensorStatus : 36|3@1+ (1,0) [0|7] ""  FC
 SG_ MotorLoopTempSensorStatus : 33|3@1+ (1,0) [0|7] ""  FC
 SG_ MotorLoopPressureSensorStatus : 39|3@1+ (1,0) [0|7] ""  FC
 SG_ MC_CoolingLoopsData : 42|4@1+ (1,0) [0|15] ""  FC
 SG_ MotorLoopPressure : 23|7@1+ (1,0) [0|127] "kPa"  FC
 SG_ InverterLoopPressure : 16|7@1+ (1,0) [0|127] "kPa"  FC
 SG_ MotorLoopFluidTemp : 8|8@1+ (1,-20) [-20|235] "�C"FCCU
 SG_ InverterLoopFluidTemp : 0|8@1+ (1,-20) [-20|235] "�C"FCCU

BO_ 514 CoolingLoopModes: 2 FC
 SG_ SpeakerRequest : 6|1@1+ (1,0) [0|1] ""  CCU
 SG_ MotorLoopPumpMode : 11|3@1+ (1,0) [0|100] ""  CCU
 SG_ MotorLoopFanMode : 8|3@1+ (1,0) [0|100] ""  CCU
 SG_ InverterLoopPumpMode : 3|3@1+ (1,0) [0|100] ""  CCU
 SG_ InverterLoopFanMode : 0|3@1+ (1,0) [0|255] ""  CCU

BO_ 257 PedalPositions: 7 PIU
 SG_ DriveButton : 32|2@1+ (1,0) [0|3] ""  FC
 SG_ Bpps1Status : 37|3@1+ (1,0) [0|7] ""  FC
 SG_ Bpps0Status : 34|3@1+ (1,0) [0|7] ""  FC
 SG_ Apps1Status : 43|3@1+ (1,0) [0|7] ""  FC
 SG_ Apps0Status : 40|3@1+ (1,0) [0|7] ""  FC
 SG_ MC_PedalPosition : 48|4@1+ (1,0) [0|15] ""  FC
 SG_ BrakePedalPosition1 : 8|8@1+ (1,0) [0|255] "%"  FC
 SG_ BrakePedalPosition0 : 0|8@1+ (1,0) [0|255] "%"  FC
 SG_ AcceleratorPedalPosition0 : 24|8@1+ (1,0) [0|255] "%"  FC
 SG_ AcceleratorPedalPosition1 : 16|8@1+ (1,0) [0|255] "%"  FC
 SG_ BspdFault : 46|1@1+ (1,0) [0|1] ""  FC



CM_ SG_ 1283 VcuErrors "Errors and warnings are represented by specific code numbers, and are reported by the FC.";
BA_DEF_  "BusType" STRING ;
BA_DEF_ BU_  "NodeLayerModules" STRING ;
BA_DEF_ BU_  "ECU" STRING ;
BA_DEF_ BU_  "CANoeJitterMax" INT 0 0;
BA_DEF_ BU_  "CANoeJitterMin" INT 0 0;
BA_DEF_ BU_  "CANoeDrift" INT 0 0;
BA_DEF_ BU_  "CANoeStartDelay" INT 0 0;
BA_DEF_DEF_  "BusType" "";
BA_DEF_DEF_  "NodeLayerModules" "";
BA_DEF_DEF_  "ECU" "";
BA_DEF_DEF_  "CANoeJitterMax" 0;
BA_DEF_DEF_  "CANoeJitterMin" 0;
BA_DEF_DEF_  "CANoeDrift" 0;
BA_DEF_DEF_  "CANoeStartDelay" 0;
BA_ "BusType" "CAN";
VAL_ 1283 VehicleStatus 2 "ON" 1 "ACCESSORY" 0 "LOCK" ;
VAL_ 513 InverterLoopTempSensorStatus 4 "SHORTED" 3 "GROUNDED" 2 "OUT_OF_RANGE_UPPER" 1 "OUT_OF_RANGE_LOWER" 0 "NO_FAULT" ;
VAL_ 513 InverterLoopPressureSensorStatus 4 "SHORTED" 3 "GROUNDED" 2 "OUT_OF_RANGE_UPPER" 1 "OUT_OF_RANGE_LOWER" 0 "NO_FAULT" ;
VAL_ 513 MotorLoopTempSensorStatus 4 "SHORTED" 3 "GROUNDED" 2 "OUT_OF_RANGE_UPPER" 1 "OUT_OF_RANGE_LOWER" 0 "NO_FAULT" ;
VAL_ 513 MotorLoopPressureSensorStatus 4 "SHORTED" 3 "GROUNDED" 2 "OUT_OF_RANGE_UPPER" 1 "OUT_OF_RANGE_LOWER" 0 "NO_FAULT" ;
VAL_ 514 SpeakerRequest 1 "ON" 0 "OFF" ;
VAL_ 514 MotorLoopPumpMode 4 "FULL" 3 "HIGH" 2 "MEDIUM" 1 "LOW" 0 "OFF" ;
VAL_ 514 MotorLoopFanMode 4 "FULL" 3 "HIGH" 2 "MEDIUM" 1 "LOW" 0 "OFF" ;
VAL_ 514 InverterLoopPumpMode 4 "FULL" 3 "HIGH" 2 "MEDIUM" 1 "LOW" 0 "OFF" ;
VAL_ 514 InverterLoopFanMode 4 "FULL" 3 "HIGH" 2 "MEDIUM" 1 "LOW" 0 "OFF" ;
VAL_ 257 DriveButton 2 "SHUTDOWN" 1 "ON" 0 "OFF" ;
VAL_ 257 Bpps1Status 4 "SHORTED" 3 "GROUNDED" 2 "OUT_OF_RANGE_UPPER" 1 "OUT_OF_RANGE_LOWER" 0 "NO_FAULT" ;
VAL_ 257 Bpps0Status 4 "SHORTED" 3 "GROUNDED" 2 "OUT_OF_RANGE_UPPER" 1 "OUT_OF_RANGE_LOWER" 0 "NO_FAULT" ;
VAL_ 257 Apps1Status 4 "SHORTED" 3 "GROUNDED" 2 "OUT_OF_RANGE_UPPER" 1 "OUT_OF_RANGE_LOWER" 0 "NO_FAULT" ;
VAL_ 257 Apps0Status 4 "SHORTED" 3 "GROUNDED" 2 "OUT_OF_RANGE_UPPER" 1 "OUT_OF_RANGE_LOWER" 0 "NO_FAULT" ;
VAL_ 257 BspdFault 1 "FAULT_ACTIVE" 0 "NO_FAULT" ;

